core/src/main/scala/ai/h2o/sparkling/backend/external/ExternalBackendConf.scala

```diff
@@ -62,8 +63,10 @@ trait ExternalBackendConf extends SharedBackendConf with Logging {
 
   def isKillOnUnhealthyClusterEnabled: Boolean = sparkConf.getBoolean(PROP_EXTERNAL_CLUSTER_KILL_ON_UNHEALTHY._1, PROP_EXTERNAL_CLUSTER_KILL_ON_UNHEALTHY._2)
 
+  @DeprecatedMethod("""getOption("spark.yarn.principal")""", "3.32")
```

Comment:
Thanks! Do we please have JIRA for removal?

Reply:
As discussed in PR to rel-3.28.1, methods won't be deprecated in the end.